packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -120,10 +139,52 @@ export class FluidCache implements IPersistedCache {
 		});
 	}
 
+	private async openDb() {
+		if (this.immediateClose) {
+			return getFluidCacheIndexedDbInstance(this.logger);
+		}
+		if (this.db === undefined) {
+			const dbInstance = await getFluidCacheIndexedDbInstance(this.logger);
+			if (this.db === undefined) {
```

Comment:
It would be great to capture somehow in telemetry how often reuse happens vs. not

Reply:
Currently recorded with FluidCacheAccess event only rather than introducing another event.